pkgs/build-support/rust/build-rust-crate/build-crate.nix

```diff
@@ -7,7 +7,7 @@
   dependencies,
   crateFeatures, crateRenames, libName, release, libPath,
   crateType, metadata, crateBin, hasCrateBin,
-  extraRustcOpts,
+  extraRustcOpts, verbose, colors,
```

Comment:
we can probably only revert this part, but I don't want to wait any longer for this